### DISCUSSION TITLE
Valtio dev tools

### DISCUSSION THREAD

**Discussion by axelson** [pmndrs/valtio#390] [Q&A]
Is there an equivalent to `useAtomsDevtools` for Valtio?

I know that there is `devtools` from `valtio/utils`: https://github.com/pmndrs/valtio/tree/2761da6a9d3083fa21a0c9d29baa774e5425904f#dev-tools

But that only allows for inspecting a single valtio proxy (similar to Jotai's `useAtomDevtools`). Is there a way to enable the devtools on all valtio proxies?

**Comment by dai-shi** [comment]
In fact, valtio is more flexible about it, thanks to module state (not react context).
You should be able to do something like this:
```js
devtools(proxy([state1, state2]))
```
